notebooks/JHU_COVID-19.ipynb

```diff
@@ -121,6 +122,49 @@
     "df[\"Date\"] = pd.to_datetime(df[\"Date\"])"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Add leading zeroes where needed"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df['FIPS'] = df['FIPS'].apply(lambda row: row.zfill(5) if not row == '' else row)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Fix County-Equivalent Entities"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df['FIPS'] = df['FIPS'].apply(lambda row: re.sub(r'^(0{3,})(\\d{2})$', r'\\g<2>\\g<1>', row))"
```

Comment:
instead of apply, we should use vectorized `str` like df['FIPS'].str.replace( ...regexp=True)